### ISSUE TITLE
Obsoleted Packet MariaDB 5.5

### ISSUE THREAD

**Issue by Marhbere** [ConSol/omd#87]
Hello guys,
I can do some work around if I don't have a fresh install and currently have Updated version of DB?

The case is:

> Package 1:mariadb-server-5.5.60-1.el7_5.x86_64 is obsoleted by mysql-community-server-5.7.27-1.el7.x86_64 which is already installed.

I Would use this server where I have somes other service.

Thanks a lot.
Marhbere.

**Comment by sni** [comment]
Which os is that? In debian and ubuntu we already support alternatives like `mariadb-server|mysql-server|virtual-mysql-server`.

**Comment by Marhbere** [comment]
Hi Sven, The OS is Centos7: CentOS Linux release 7.6.1810 (Core) 

uname -a:
`Linux localhost.localdomain 3.10.0-862.3.3.el7.x86_64 #1 SMP Fri Jun 15 04:15:27 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
`

Installed omd from: 
` rpm -Uvh "https://labs.consol.de/repo/stable/rhel7/i386/labs-consol-stable.rhel7.noarch.rpm"`

When I try install running: `yum install omd ` it break with the output;

```
--> Processing Dependency: mariadb-server for package: omd-3.10-labs-edition-el7-1.x86_64
Package 1:mariadb-server-5.5.60-1.el7_5.x86_64 is obsoleted by mysql-community-server-5.7.27-1.el7.x86_64 which is already installed
--> Finished Dependency Resolution
Error: Package: omd-3.10-labs-edition-el7-1.x86_64 (labs_consol_stable)
           Requires: mariadb-server
           Available: 1:mariadb-server-5.5.60-1.el7_5.x86_64 (base)
               mariadb-server = 1:5.5.60-1.el7_5
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```


Regards
